build.gradle

```diff
@@ -45,16 +43,16 @@ dependencies {
     // For the testRepo tests I am locking the versions, instead of a open version, as it makes
     // unit testing easier, This does not affect the final artifact.
     // If you change values here, you need to update JRubyJarPluginSpec as well.
-    testRepo ('org.jruby:jruby-complete:1.7.17') {
+    testRepo ('org.jruby:jruby-complete:1.7.19') {
+        transitive = false
+    }
+    testRepo ('de.saumya.mojo:jruby-mains:0.2.0') {
         transitive = false
     }
     testRepo ("org.spockframework:spock-core:0.7-groovy-2.0") {
         transitive = false
     }
 
-    testRepo 'com.github.jengelman.gradle.plugins:shadow:1.1.2'
-
-}
```

Comment:
It looks like the removal of this final closing `}` ius causing the build to fail in Travis